src/main/java/org/fcrepo/spec/testsuite/versioning/LdprvHttpPut.java

```diff
@@ -96,17 +135,22 @@ public void ldpnrvMustSupportPUT(final String uri) {
      *
      * @param uri The repository root URI
      */
-    //@Test(groups = {"MUST"})
+    @Test(groups = {"MUST"})
     @Parameters({"param1"})
     public void ldpnrvMustSupportPUTForExistingResources(final String uri) {
         final TestInfo info = setupTest("4.1.2-D", "ldpnrvMustSupportPUTForExistingResources",
                                         "Must support PUT for updating existing  LDPNRvs",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldprv-put",
                                         ps);
         //create an LDPNRv
+        final Response response = putVersionedResourceWithBodyUnverified(uri, info, "test");
+        assertEquals(response.getStatusCode(), 201);
+        final String resourceUri = getLocation(response);
+        final Response getResponse = doGet(resourceUri);
         //verify that "Allow: PUT" header is present
+        confirmPresenceOfHeaderValueInMultiValueHeader("Allow", "PUT", getResponse);
         //update an existing LDPNRv using a put
-        //verify that it is a TimeGate and has a TimeMap
-
+        final Response response2 = putVersionedResourceWithBodyUnverified(resourceUri,"test2");
+        assertEquals(response2.getStatusCode(), 204);
```

Comment:
For completeness, we may as well finish this test with `GET` on the resource to verify that the content has indeed been updated... along the lines of: https://github.com/fcrepo/Fedora-API-Test-Suite/pull/149/files#diff-8817daed51e1756ffb430de8b1f36ae8R105